packages/org-unit-tree/src/OrgUnitTree.component.js

```diff
@@ -369,6 +381,7 @@ OrgUnitTree.defaultProps = {
     hideCheckboxes: false,
     hideMemberCount: false,
     orgUnitsPathsToInclude: null,
+    foreceReloadChildren: false,
```

Comment:
@HendrikThePendric 
a little typo in `foreceReloadChildren` here and above 😉

Reply:
Oh shoot. Thanks for pointing that out. Will fix.